docs/Extended_dry_run.rst

```diff
@@ -0,0 +1,731 @@
+.. _extended_dry_run:
+
+Extended dry run
+================
+
+Using ``--extended-dry-run`` or ``-x`` (supported since EasyBuild v2.4.0), a detailed overview of the build and install
+procedure that EasyBuild is going to execute can be obtained almost instantly.
+
+All time-consuming operations, including executing commands to configure/build/install the software,
+are only *reported* rather than being actually performed.
+
+Example output is available at :ref:`extended_dry_run_examples`.
+
+.. contents::
+    :depth: 3
+    :backlinks: none
+
+.. _extended_dry_run_notes:
+
+Important notes
+---------------
+
+There are a couple of things you should be aware of when using ``--extended-dry-run`` and interpreting the output it
+produces.
+
+.. _extended_dry_run_notes_differences:
+
+Build/install procedure reported by dryn run may be (slightly) different
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The actual build and install procedure may (slightly) differ from the one reported by ``--extended-dry-run``,
+due to conditional checks in the easyblock being used.
+
+For example, expressions that are conditional on the presence of certain files or directories in the build directory
+will always be false, since the build directory is never actually populated.
+
+.. _extended_dry_run_notes_ignored_errors:
+
+Errors are ignored (by default) during dry run
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Any errors that occur are ignored, and are reported with a clear warning message.
+This is done because it is not unlikely that these errors occur because of the dry run mechanism.
```

Comment:
double negative is harder to read. Perhaps change to "likely" or "possible" or "probable"